code/datums/station_traits/positive_traits.dm

```diff
@@ -333,3 +333,29 @@
 		return
 	var/obj/item/organ/internal/cybernetic = new cybernetic_type()
 	cybernetic.Insert(spawned, special = TRUE, drop_if_replaced = FALSE)
+
+/datum/station_trait/glass_maint
+	name = "Glass Maintenance"
+	trait_type = STATION_TRAIT_POSITIVE
+	show_in_report = TRUE
+	weight = 2
+
+/datum/station_trait/glass_maint/New()
+	. = ..()
+	report_message = pick("In an effort to reduce maintenance casualties, glass windows have been fitted in every door.",
+		"We had a spare engineer, so we had them put glass in all maintenance doors.",
+		"To commemorate our miners, we put glass in all the airlocks. Unfortunately, we ran out after completing maintenance.",
+		"In an effort to comply with Space Law section [rand(1,99999)].4, we have outfitted maintenence with glass windows.",
+		"An anomaly caused all the maintenance airlocks to get glass windows.",
+		"An intern went through and added glass to all the maintenance windows in their spare time.",
+		"In an effort to promote brand transparancy, we're adding glass to all maintenance doors effective immediately.",  // my proudest dad joke
+		"We over-ordered glass during construction, so maintenance gets glass windows.")
```

Comment:
```suggestion
	report_message = pick(
		"In an effort to reduce maintenance casualties, glass windows have been fitted in every door.",
		"We had a spare engineer, so we had them put glass in all maintenance doors.",
		"To commemorate our miners, we put glass in all the airlocks. Unfortunately, we ran out after completing maintenance.",
		"In an effort to comply with Space Law section [rand(1,99999)].4, we have outfitted maintenance with glass windows.",
		"An anomaly caused all the maintenance airlocks to get glass windows.",
		"An intern went through and added glass to all the maintenance windows in their spare time.",
		"In an effort to promote brand transparency, we're adding glass to all maintenance doors effective immediately.",  // my proudest dad joke
		"We over-ordered glass during construction, so maintenance gets glass windows.",
	)
```

Reply:
fixes typos and also follows the multithreading rules

Reply:
i tried that originally and it spat a compile error

Reply:
try it again